src/ontology/config/prefixes.csv

```diff
@@ -224,8 +224,19 @@ PLANP,http://purl.obolibrary.org/obo/PLANP_
 CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
+ICD10CM2,https://icd.codes/icd10cm/
```

Comment:
## New entries to `prefixes.csv`
Addresses [this issue](https://github.com/monarch-initiative/mondo-ingest/pull/363#discussion_r1327876041).

@matentzn You may want to look over these prefixes.
Some were missing, and I simply added.
But some are duplicates of other prefixes with different URI stems:
- `ICD10WHO` & `ICD10WHO2010`
- `UMLS` & `UMLS2`
- `OMIMPS` & `OMIMPS2` (due to [this issue](https://github.com/monarch-initiative/omim/issues/103))

Reply:
## Filter any namespaces out?
Should I be outputting all namespaces, or filter just to leave the ones ingested by `mondo-ingest` (e.g. DO, ORDO, etc)?

Reply:
In addition to doing what's needed with extended prefix maps and using that, we want to handle prefixes.csv like so:
@hrshdhgd Nico and I discussed, and the standard format for prefix maps will be `yaml`. I'll write a util function that converts that to `prefixes.csv` so that we can use that with SemanticSQL.

Reply:
Not a concern anymore / currently for this PR.

Moved this task to new issue:
- https://github.com/monarch-initiative/mondolib/issues/8